balancer/src/main/resources/reference.conf

```
@@ -1,11 +1,15 @@
-com.olegych.scastie.balancer {
+org.scastie.balancer {
   snippets-storage = files
   snippets-dir = ./target/snippets/
   old-snippets-dir = ./target/old-snippets/
 
-  remote-hostname = "127.0.0.1"
+  remote-sbt-hostname = "127.0.0.1"
   remote-sbt-ports-start = 5150
   remote-sbt-ports-size = 1
+
+  remote-scli-hostname = "127.0.0.1"
+  remote-scli-ports-start = 5250
+  remote-scli-ports-size = 1
```

remote hostname is localhost :stuck_out_tongue_closed_eyes: 
```suggestion
  sbt-hostname = "127.0.0.1"
  sbt-ports-start = 5150
  sbt-ports-size = 1

  scli-hostname = "127.0.0.1"
  scli-ports-start = 5250
  scli-ports-size = 1
```